releases/latest/edge/mlflow/bundle.yaml

```diff
@@ -1,5 +1,6 @@
 bundle: kubernetes
 name: mlflow
+docs: https://discourse.charmhub.io/t/mlflow-docs-index/10836
```

Comment:
The link looks OK, but let's update the structure after we merge this PR to align more with the initial gdoc we have for the Buldne docs

Reply:
Good idea :+1: